site/content/docs/user/quick-start.md

```diff
@@ -347,3 +363,4 @@ kind, the Kubernetes cluster itself, etc.
 [Kubernetes imagePullPolicy]: https://kubernetes.io/docs/concepts/containers/images/#updating-images
 [Private Registries]: /docs/user/private-registries
 [customize control plane with kubeadm]: https://kubernetes.io/docs/setup/independent/control-plane-flags/
+[docker enable ipv6]: https://docs.docker.com/config/daemon/ipv6/
```

Comment:
the instructions on that page don't set `fixed-cidr-v6` so the daemon fails to start

Reply:
I remember running into this now... shall we send them a PR? 🙃 
https://github.com/docker/docker.github.io/edit/master/config/daemon/ipv6.md

Reply:
They have it documented here https://docs.docker.com/v17.09/engine/userguide/networking/default_network/ipv6/ , don´t know why they didn´t explain it in that link